components/table/html/_mixin.scss

```diff
@@ -17,6 +17,10 @@
     @include utrecht-table__caption;
   }
 
+  figcaption {
+    @include utrecht-table__figcaption;
+  }
+
```

Comment:
This would style every `figcaption` this way, that would be undesirable.

So I think we cannot support `figcaption` for tables specifically in the HTML components. My suggestion is to remove this code.